src/windows/about.py

```diff
@@ -75,8 +75,7 @@ def __init__(self):
                                 self.btnchangelog.setVisible(True)
                                 break
                     except:
-                        # Ignore decoding errors
-                        pass
+                        log.warning('Failed to parse log file %s with encoding %s' % (changelog_path, encoding_name))
```

Comment:
`log.info()` at _most_, preferably `log.debug() even (so the users won't see it)`. There is nothing the user can do about that, and no reason they should care.

Reply:
Oh, and this is another one where it needs to be `except Exception:` at a minimum.